spring-velocity-support/src/main/java/org/apache/velocity/spring/SpringResourceLoader.java

```diff
@@ -91,8 +91,8 @@ public void init(ExtProperties configuration) {
     		}
     	}
     	if (logger.isInfoEnabled()) {
-    		logger.info("SpringResourceLoader for Velocity: using resource loader [" + this.resourceLoader +
-    				"] and resource loader paths " + Arrays.asList(this.resourceLoaderPaths));
+    		logger.info("SpringResourceLoader for Velocity: using resource loader [{}] and resource loader paths {}",
+    		        resourceLoader, Arrays.asList(this.resourceLoaderPaths));
```

Comment:
There is a mix of tabs and spaces for indentation. We use four spaces. Is it possible to fix this?

Reply:
Yes, I'll fix that.